### DISCUSSION TITLE
DSEP Working Group | Call 18 | July 06, 2022

### DISCUSSION THREAD

**Discussion by sujeets-protean** [beckn/DSEP-Specification#45] [General]
Agenda
======
1 | Use case mapping activity | 40 mins
2 | Open discussion | 20 mins
3 | Action Items | 10 mins

Deck and Recording to follow.

**Comment by sujeets-protean** [comment]
Deck
=========
[DSEP Working Group - Call 18 - July 06, 2022.pdf](https://github.com/beckn/DSEP-Specification/files/9054339/DSEP.Working.Group.-.Call.18.-.July.06.2022.pdf)

@core-wg-admin please put call recording here

**Reply by Anand-EkStep** [reply]
Here is the recording:
https://drive.google.com/file/d/1PxUxwGbWhRCFq6ZWdiMsMVE_Ww4mu0YP/view?usp=sharing
